src/Psalm/Internal/Analyzer/Statements/BreakAnalyzer.php

```diff
@@ -16,7 +16,7 @@
 class BreakAnalyzer
 {
     public static function analyze(
-        StatementsAnalyzer $statements_analyzer,
+        StatementsAnalyzer $_statements_analyzer,
```

Comment:
we're on master, if it's not used anymore, we could remove it

Reply:
I thought about it, but I wondered if it'd be better left for consistency since all of the calls [here](https://github.com/AndrolGenhald/psalm/blob/try-analyzer-rewrite/src/Psalm/Internal/Analyzer/StatementsAnalyzer.php#L533:L580) are otherwise the same though. It's all internal I guess, so might as well remove it.